util/winutil/winutil.go

```diff
@@ -30,27 +33,45 @@ func GetDesktopPID() (pid uint32, ok bool) {
 	return pid, pid != 0
 }
 
-// GetRegString looks up a registry path in our local machine path, or returns
-// the given default if it can't.
+// GetPolicyString looks up a registry value in our local machine's path for
+// system policies, or returns the given default if it can't.
 //
 // This function will only work on GOOS=windows. Trying to run it on any other
 // OS will always return the default value.
-func GetRegString(name, defval string) string {
-	key, err := registry.OpenKey(registry.LOCAL_MACHINE, RegBase, registry.READ)
+func GetPolicyString(name, defval string) string {
+	s, err := getRegString(regPolicyBase, name)
 	if err != nil {
-		log.Printf("registry.OpenKey(%v): %v", RegBase, err)
-		return defval
+		// Fall back to the legacy path for policies
+		return GetRegString(name, defval)
```

Comment:
pretty confusing having two funcs called here in GetPolicyString both called "get reg string" but with different case and different signatures.

It took me a few reads to figure out what was going on.

Should we rename some things and get "OrDefault" in the name of one?

Reply:
I think `GetRegString` and `GetRegInteger` can just be removed entirely and we can migrate all callers to the Policy versions. Then the existing behavior can simply be included in the Policy functions.

Reply:
Yeah, I'll do something about the naming, however I do not want to get rid of the non-policy variants, as is is not always desirable to expose all registry settings to admin policy.